### PR TITLE
fix(curriculum): Minor Reword of Generate Random Whole Numbers with JavaScript

### PR DESCRIPTION
**Pull request by jeremylt**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

See discussion in: https://forum.freecodecamp.org/t/basic-javascript-generate-random-whole-numbers-with-javascript/605101

### CODE REVIEW

**Review by Sembauke — APPROVED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/generate-random-whole-numbers-with-javascript.md [hunk 1]
```diff
@@ -9,23 +9,27 @@ dashedName: generate-random-whole-numbers-with-javascript
 
 # --description--
 
-It's great that we can generate random decimal numbers, but it's even more useful if we use it to generate random whole numbers.
+We can generate random decimal numbers with `Math.random()`, but sometimes we need to generate random whole numbers. The following process will give us a random whole number less than `20`:
```
Comment (Sboonny) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/generate-random-whole-numbers-with-javascript.md:12]:
```suggestion
You can generate random decimal numbers with `Math.random()`, but sometimes you need to generate random whole numbers. The following process will give you a random whole number less than `20`:
```

**Review by Sboonny — COMMENTED** [review]
The idea LGTM, some nitpicks in the original file itself.

- the list can be in Markdown, no benefit in having it as HTML
- if changing the files, the use of the word "we" can be changed to "you", as well.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Comment by lasjorg** [comment]
@jeremylt This is way better now, in my opinion. Thank you for the PR. 👍

**Pull request merged**